File: src/main/java/com/github/stanislavmikheyev/pgsgradessource/entity/Student.java
package com.github.stanislavmikheyev.pgsgradessource.entity;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
public class Student {
    @Id
    private Long id;

    @Column
    private String name;

    @Column
    private String surname;
}

File: src/main/kotlin/com/tighub/stanislavmikheyev/pgsgradessource/entity/Student.kt
package com.tighub.stanislavmikheyev.pgsgradessource.entity

import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.Id

@Entity
data class Student(
    @Id
    val id: Long,

    @Column
    var name: String,

    @Column
    val surname: String
)
